### ISSUE TITLE
Mac v2.14 Install issue on Sonoma 14.3

### ISSUE THREAD

**Issue by ryanniuu** [cxasm/notepad--#339]
can load dmg OK
but system won't allow install, before getting to security/permission setting

Error Message:
"Notepad--.app" is damaged and can't be opened. You should eject the disk image.
and only allow me to "Eject Disk Image"

Tried moving the app to Applications folder manually and open from there, same error
Tried open with "ctrl" but same issue

**Comment by cxasm** [comment]
There is a problem with it. NDD has no signature under macos. Because I failed to register for a macos developer account. There is a high probability of audit failure in China.

**Comment by ryanniuu** [comment]
thanks for the update
its used to be fairly easy to register apple developer account; 
look forward to next release with fix to this

**Comment by cxasm** [comment]
No, it's easy to fail to register an Apple developer account in China, with a probability of at least 50% or more. And once it fails, I won't be able to successfully register again in the future unless I completely change my device and ID, which has tormented me.

And it's also very difficult to get on GitHub in China, almost impossible. I need to refresh 10 times to reply to a post. You don't know about the Internet in China.

**Comment by dfemq** [comment]
Launch Terminal and then issue the following command:

xattr -cr /path/to/notepad--.app
